File: Py Scripts/CountDown.py
import UnityEngine

class CountDown(UnityEngine.MonoBehaviour):
    def Start(self):
        self.gameControl = UnityEngine.Object.FindObjectOfType(GameControl)
        self.animator = self.GetComponent(UnityEngine.Animator)

    def StartCountdown(self):
        self.animator.SetTrigger("StartCountdown")

    def StartGame(self):
        self.gameControl.StartGame()

File: scripts/CountDown.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class CountDown : MonoBehaviour
{
    private GameControl gameControl;
    private Animator animator;
    // Start is called before the first frame update
    void Start()
    {
        gameControl = FindObjectOfType<GameControl>();
        animator = GetComponent<Animator>();
    }

    public void StartCountdown()
    {
        animator.SetTrigger("StartCountdown");
    }

    public void StartGame()
    {
        gameControl.StartGame();
    }
}
